methcomp/regression.py

```diff
@@ -1,532 +1,279 @@
+        Amount of bootstrap estimates that should be performed to acquire
+        standard errors (and confidence intervals). If None, no bootstrap
+        is performed.
+        [default=1000]
     x_label : str, optional
-        The label which is added to the X-axis. If None is provided, a standard
-        label will be added.
+        The label which is added to the X-axis. (default: "Method 1")
     y_label : str, optional
-        The label which is added to the Y-axis. If None is provided, a standard
-        label will be added.
+        The label which is added to the Y-axis. (default: "Method 2")
     title : str, optional
-        Title of the plot. If None is provided, no title will be plotted.
-    CI : float, optional
-        The confidence interval employed in Deming line. Defaults to 0.95.
+        Title of the regression plot. If None is provided, no title will be plotted.
     line_reference : bool, optional
-        If True, a grey reference line at y=x will be plotted in the plot.
-        Defaults to true.
+        If True, a grey reference line at y=x will be plotted in the plot
+        (default: True)
     line_CI : bool, optional
         If True, dashed lines will be plotted at the boundaries of the confidence intervals.
-        Defaults to false.
+        (default: False)
     legend : bool, optional
-        If True, will provide a legend containing the computed Deming equation.
-        Defaults to true.
-    color_points : str, optional
-        Color of the individual differences that will be plotted.
-        Color should be provided in format compatible with matplotlib.
-    color_deming : str, optional
-        Color of the mean difference line that will be plotted.
-        Color should be provided in format compatible with matplotlib.
+        If True, will provide a legend containing the computed regression equation.
+        (default: True)
     square : bool, optional
         If True, set the Axes aspect to "equal" so each cell will be
-        square-shaped.
-    point_kws : dict of key, value mappings, optional
-        Additional keyword arguments for `plt.scatter`.
-    ax : matplotlib Axes, optional
-        Axes in which to draw the plot, otherwise use the currently-active
-        Axes.
+        square-shaped. (default: True)
+    ax : matplotlib.axes.Axes, optional
+        matplotlib axis object, if not passed, uses gca()
+    point_kws : Optional[Dict], optional
+        Additional keywords to plt
+    color_regr : Optional[str], optional
+        color for regression line and CI area
+    alpha_regr : Optional[float], optional
+        alpha for regression CI area
 
     Returns
     -------
-    ax : matplotlib Axes
-        Axes object with the Deming plot.
+    matplotlib.axes.Axes
+        axes object with the Deming regression plot
 
     See Also
     -------
-    Koopmans, T. C. (1937). Linear regression analysis of economic time series. DeErven F. Bohn, Haarlem, Netherlands.
```

Comment:
What's the reason you want the scientific references to be deleted?

Reply:
Oops. I intended them to be under:

References
-----------
Fixed

Reply:
Resolved.